content/v2.0/reference/cli/influx/config/create.md

```diff
@@ -0,0 +1,26 @@
+---
+title: influx config create
+description: The 'influx config create' command creates a new InfluxDB connection configuration.
+menu:
+  v2_0_ref:
+    name: influx config create
+    parent: influx config
+weight: 201
+---
+
+The `influx config create` command creates a new InfluxDB connection configuration in the `config` file (by default, stored at `~/.influxdbv2/config`).
```

Comment:
...creates a new InfluxDB connection configuration **and stores it** in the `config` file (by default, stored at `~/.influxdbv2/config`).